velox/dwio/parquet/writer/Writer.h

```diff
@@ -94,6 +94,7 @@ struct WriterOptions {
   // The default factory allows the writer to construct the default flush
   // policy with the configs in its ctor.
   std::function<std::unique_ptr<DefaultFlushPolicy>()> flushPolicyFactory;
+  ::arrow::util::CodecOptions codecOptions;
```

Comment:
you're missing an include or something. Check the CI failure.

Reply:
I add the included header and run success on my machine